modules/lite/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@deck.gl/lite",
   "description": "Scripting with deck.gl",
   "license": "MIT",
-  "version": "5.2.0-alpha.1",
+  "version": "0.0.1",
```

Comment:
I wish the JSON format would allow you to put a comment here to explain why we are using a different numbering scheme here... 

If this module doesn't publish, set to 0.0.0 or drop the version line altogether?

Reply:
I tried to remove it and it results in `yarn` not installing this module's dependencies. I just want to change this so that it's absolutely clear that this version number is not used. Will add a comment.